pyaccel/accelerator.py

```diff
@@ -192,7 +191,7 @@ def vchamber_on(self, value):
     @property
     def lattice_version(self):
         """Return lattice version."""
-        return self._lattice_version
+        return self.trackcpp_acc.lattice_version
```

Comment:
is this a python string, or it is a CPP string object? In case it is the second option, you'll need to convert it.